src/Spec2-Dialogs/SpRequestDialog.class.st

```diff
@@ -168,9 +168,15 @@ SpRequestDialog >> initializePresenters [
 
 { #category : 'initialization' }
 SpRequestDialog >> initializeWindow: aWindowPresenter [
-	
+
 	super initializeWindow: aWindowPresenter.
-	aWindowPresenter whenOpenedDo: [ textInput takeKeyboardFocus; selectAll  ]
+	self preferredExtent: 400@200.
```

Comment:
why not doing `self preferredExtent: self class defaultPreferredExtent`?
I also wonder if defaultPreferredExtent is used by default.